src/applications/claims-status/utils/evidenceDictionary.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Link } from 'react-router-dom-v5-compat';
+
+export const evidenceDictionary = {
+  '21-4142/21-4142a': {
+    longDescription: (
+      <>
+        <p>
+          For your benefits claim, we need your permission to request your
+          personal information from a non-VA source, like a private doctor or
+          hospital.
+        </p>
+        <p>Personal information may include:</p>
+        <ul className="bullet-disc">
+          <li>Medical treatments</li>
+          <li>Hospitalizations</li>
+          <li>Psychotherapy</li>
+          <li>Outpatient care</li>
+        </ul>
+      </>
+    ),
+    nextSteps: (
+      <>
+        <p>
+          Use VA Form 21-4142 to give us permission to request your personal
```

Comment:
I don't think we need to do anything about this now, but I am curious to see how many subsequent evidence request updates might be using this same/similar language - maybe eventually it would make sense to make this into a template and fill in the form names via function or some such.  Food for thought for later; since we only have this one instance right now, I think this is fine for the time being.